### ISSUE TITLE
Minimap crashes with Tree Sitter parsers

### ISSUE THREAD

**Issue by robertrossmann** [atom-minimap/minimap#652]
1. Install Atom Beta 1.25-beta.1
1. Enable experimental Tree-Sitter parsers
1. Restart Atom Beta
1. Open a JavaScript file with the built-in `JavaScript` grammar (not `language-babel`)
1. Observe crash

**Atom**: 1.25.0-beta1 x64
**Electron**: 1.7.11
**OS**: Mac OS X 10.13.3
**Thrown From**: [minimap](https://github.com/atom-minimap/minimap) package 4.29.7

### Debugging info

The `.onDidTokenize()` function is not present on the `tokenizedBuffer` object, because it's an instance of `TreeSitterLanguageMode` which does not implement this event and is most likely not the object type Minimap expects to receive. I am unsure if this is a bug in Atom/Tree Sitter or if the function is missing intentionally. 🤔

This seems to have been fixed in Atom (not released yet) in https://github.com/atom/atom/pull/16744. The PR's title implies the `.onDidTokenize()` function is private and the function in Atom Beta will do _nothing_, so I wonder if it's usage could be removed from Minimap?

### Stack Trace

Failed to activate the minimap package

```
At tokenizedBuffer.onDidTokenize is not a function

TypeError: tokenizedBuffer.onDidTokenize is not a function
    at /packages/minimap/lib/minimap.js:277:30)
    at Main.minimapForEditor (/packages/minimap/lib/main.js:327:17)
    at subscriptions.add.atom.workspace.observeTextEditors (/packages/minimap/lib/main.js:390:26)
    at Workspace.observeTextEditors (/Applications/Atom Beta.app/Contents/Resources/app/src/workspace.js:637:59)
    at Main.initSubscriptions (/packages/minimap/lib/main.js:389:43)
    at Main.toggle (/packages/minimap/lib/main.js:184:12)
    at Main.activate (/packages/minimap/lib/main.js:115:55)
    at Package.activateNow (/Applications/Atom Beta.app/Contents/Resources/app/src/package.js:230:33)
    at measure (/Applications/Atom Beta.app/Contents/Resources/app/src/package.js:206:33)
    at Package.measure (/Applications/Atom Beta.app/Contents/Resources/app/src/package.js:88:25)
    at activationPromise.Promise (/Applications/Atom Beta.app/Contents/Resources/app/src/package.js:200:20)
    at Promise (<anonymous>)
    at Package.activate (/Applications/Atom Beta.app/Contents/Resources/app/src/package.js:198:38)
    at PackageManager.activatePackage (/Applications/Atom Beta.app/Contents/Resources/app/src/package-manager.js:695:42)
    at config.transactAsync (/Applications/Atom Beta.app/Contents/Resources/app/src/package-manager.js:670:36)
    at Config.transactAsync (/Applications/Atom Beta.app/Contents/Resources/app/src/config.js:852:28)
    at PackageManager.activatePackages (/Applications/Atom Beta.app/Contents/Resources/app/src/package-manager.js:668:23)
    at PackageManager.activate (/Applications/Atom Beta.app/Contents/Resources/app/src/package-manager.js:647:50)
    at loadStatePromise.loadState.then (/Applications/Atom Beta.app/Contents/Resources/app/src/atom-environment.js:867:27)
    at <anonymous>
```

### Commands

```
```

### Non-Core Packages

```
atom-chai-snippets 0.3.0 
atom-ide-ui 0.9.1 
atom-material-ui 2.1.3 
atom-ternjs 0.18.3 
autocomplete-json 5.5.2 
autocomplete-modules 1.11.0 
blame 1.0.1 
build 0.70.0 
build-dockerfile 1.0.1 
build-make 0.13.0 
docblockr 0.13.6 
editorconfig 2.2.2 
file-icons 2.1.16 
firestore-rules-grammar 0.3.0 
git-plus 7.10.0 
git-projects 1.17.0 
go-debug 1.6.0 
go-plus 5.8.2 
gruvbox-plus-syntax 1.3.1 
highlight-selected 0.13.1 
hyperclick 0.0.0 
ide-css 0.2.0 
ide-docker 0.0.2 
ide-go 0.6.1 
ide-json 0.2.1 
ide-typescript 0.7.3 
ide-yaml 0.4.2 
indent-guide-improved 1.4.13 
js-hyperclick 1.13.0 
language-babel 2.84.0 
language-diff 0.7.0 
language-docker 1.1.8 
language-dotenv 1.1.1 
language-gitignore 0.3.0 
language-graphql 0.9.0 
language-ini 1.19.0 
language-terraform 0.8.1 
linter-eslint 8.4.1 
linter-markdown 5.2.0 
linter-package-json-validator 2.0.0 
linter-shellcheck 1.4.6 
linter-terraform-syntax 1.2.2 
minimap 4.29.7 
minimap-cursorline 0.2.0 
minimap-git-diff 4.3.1 
minimap-linter 2.1.3 
platformio-ide-terminal 2.8.0 
pristine-ui 1.1.11 
pure-syntax 0.3.0 
sort-lines 0.18.0 
standardjs-snippets 2.6.2 
sync-settings 0.8.3 
teletype 0.8.0 
todo-show 2.2.0 
tree-view-autoresize 2.3.0 
```

**Comment by Evpok** [comment]
Dupe of #649, fixed in #650. Now this just needs some maintainer attention.

**Comment by robertrossmann** [comment]
Ah thanks. I’ll close this and subscribe to the PR for updates!
